tests/fixtures/virt/migration.py

```
@@ -0,0 +1,49 @@
+    vmim = []
+    for migration_index in range(request.param):
+        migration_obj = VirtualMachineInstanceMigration(
+            name=f"{vm_for_migration_test.name}-{migration_index}",
+            namespace=vm_for_migration_test.namespace,
+            vmi_name=vm_for_migration_test.vmi.name,
+            teardown=False,
+        )
+        migration_obj.deploy(wait=True)
+        migration_obj.wait_for_status(status=migration_obj.Status.SUCCEEDED, timeout=TIMEOUT_3MIN)
+        vmim.append(migration_obj)
+        LOGGER.info(f"Migration #{migration_index + 1} done")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)